packages/analytics-js-integrations/src/integrations/AdobeAnalytics/util.js

```diff
@@ -1,16 +1,14 @@
 /* eslint-disable no-param-reassign */
 /* eslint-disable no-undef */
-import get from 'lodash.get';
 import each from '@ndhoule/each';
 import { isDefined } from '@rudderstack/analytics-js-common/utilities/checks';
 import { DISPLAY_NAME } from '@rudderstack/analytics-js-common/constants/integrations/AdobeAnalytics/constants';
+import { path } from 'ramda';
 import Logger from '../../utils/logger';
-import {
-  toIso,
-  getHashFromArray,
-  isDefinedAndNotNullAndNotEmpty,
-} from '../../utils/commonUtils';
+import { toIso, getHashFromArray, isDefinedAndNotNullAndNotEmpty } from '../../utils/commonUtils';
 
+
+const get = (context, value) => path(value.split('.'), context);
```

Comment:
Have we ensured this works identical to the lodash.get? I see no tests for it, though.

Reply:
I have tested and didn't find any differences. I have added test case at `packages/analytics-js-integrations/__tests__/integrations/AdobeAnalytics/util.test.js`, it will be called with `handleLists` tests.

Reply:
Although, I'd recommend its own test suite for future maintainability.

Reply:
Yes added that.